lib/refresh.js

```diff
@@ -26,11 +26,11 @@ exports.do = (refreshToken, accessToken, apiUrl) => {
         return
       }
 
-      if (!body.refreshToken) {
+      if (!body.refresh) {
```

Comment:
@fgimenez I guess, this and the next if blocks can be removed: if result is 200 `refresh` and `access` fields are guaranteed to be present in the response body. Otherwise it won't be 200 result, and it will throw before reaching these ifs.